test/linalg/svd.jl

```diff
@@ -4,6 +4,31 @@ using Base.Test
 
 using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
+@testset "Simple svdvals / svdfact tests" begin
+    ≈(x,y) = isapprox(x,y,rtol=1e-15)
```

Comment:
In particular, `@test x ≈ y atol=1e-15` should do what you want without redefining `≈`